src/model/utils.js

```diff
@@ -39,5 +63,46 @@ export function addNewGeoDataToCache(data) {
       get(dataByGeography).set(key, new Map(value));
     }
   });
-  newDataByGeography.notify();
+  newDataByGeography.setTrue();
+}
+
+export function catAndTotalCodesRequested(requestedCategories) {
```

Comment:
I think it would be clearer if this was named in a way that made it more obvious it returns a boolean (AreCatAndTotalsCodesRequested)